terra-jupyter-base/Dockerfile

```diff
@@ -1,26 +1,15 @@
 # adapted from https://hub.docker.com/r/jupyter/base-notebook/ AKA https://github.com/jupyter/docker-stacks/tree/master/base-notebook
 
-# this is pushed to dockerhub as jmthibault79/leonardo-jupyter-min:joel-tmp-20190809-9
-
-FROM debian:stretch
-
+FROM marketplace.gcr.io/google/ubuntu1804:latest
```

Comment:
Interesting, using a Google Cloud Marketplace base image. Seems reasonable to me.

I'm slightly concerned about using the `latest` tag -- I wonder if we should tie to a particular hash to avoid unexpected upstream changes. Or is that not likely if we're staying on the same version of Ubuntu?

Reply:
Bernick suggested we start using google maintained base images for security concerns...I wasn't able to find other tags for this image, since we're prebuilding these docker images, so things shouldn't all of sudden blow up in prod at least. And it's probably good to keep up-to-date base ubuntu image.

Reply:
Yeah, that makes sense 👍